geonode/templates/base.html

```diff
@@ -29,7 +29,68 @@
       <link href="{% static "geonode/css/ux-all.css" %}?v={{ VERSION }}" rel="stylesheet" />
       <link href="{% static "geonode/css/base.css" %}?v={{ VERSION }}" rel="stylesheet" />
       <style type="text/css">[ng\:cloak],[ng-cloak],[data-ng-cloak],[x-ng-cloak],.ng-cloak,.x-ng-cloak,.ng-hide:not(.ng-hide-animate){display:none !important;}</style>
+      <style type="text/css">
+        /* Additional styles by custom themes */
+        {% if custom_theme.body_color %}
+        body {
+          background: {{ custom_theme.body_color }};
+        }
+        {% endif %}
+
+        {% if custom_theme.navbar_color %}
+        .home .navbar-inverse , .navbar-inverse {
+          background-color: {{ custom_theme.navbar_color }};
+        }
+        {% endif %}
+
+        {% if custom_theme.logo %}
+        .navbar-brand {
+          background-image: url({{ custom_theme.logo.url }});
```

Comment:
add to .navbar-brand also the following css in order to correctly fit the logo on the nav-bar

```
background-repeat: no-repeat;
background-position: center;
background-size: contain;
```

Reply:
`base.less` has this already :
```
.navbar-brand {
	background: url(../img/logo.png) 0 10px no-repeat;
	background-size: 100%;
	text-indent: -9999px;
	height: 65px;
	width: 200px;
}
```
So we should be good... Or did you have some logo that don't fit properly ? (on my tests, logo displays correctly)

Reply:
Sorry just tested again, you're right.